hacs.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "Raspberry Pi GPIO",
-  "homeassistant": "2022.7.0"
+  "homeassistant": "2024.9.0"
```

Comment:
I think you need to update your fork, that is already 2024.9.0 on main.
Btw, you can directly work on this repo (no need to work on a fork) if you want so you don't need to constantly update your fork.